ecosystem/cli/website.py

```diff
@@ -22,6 +23,10 @@ def __init__(self, root_path: Optional[str] = None):
         self.current_dir = root_path or os.path.abspath(os.getcwd())
         self.resources_dir = "{}/ecosystem/resources".format(self.current_dir)
         self.dao = DAO(path=self.resources_dir)
+        with open(self.resources_dir + "/labels.json") as file:
+            self.label_descriptions = {
+                item["name"]: item["description"] for item in json.load(file)
+            }
```

Comment:
pathlib is much nicer. You could extract a helper variable for the `Path().read_text()` part if the below is too dense.

```suggestion
        self.label_descriptions = {
                item["name"]: item["description"]
                for item
                in json.loads(Path(self.resources_dir, "labels.json").read_text())
            }
```